Jenkinsfile.github

```diff
@@ -56,29 +56,8 @@ pipeline {
                         rm -fr dist
                         make build/.env
                     """
-                }
-            }
-        }
-
-        stage('Pre-Release Validate') {
-            when { tag "v*" }
-            parallel {
-                stage('Validate docs-csm tag') {
-                    steps {
-                        script {
-                            // verify head of correlated to DOCS-CSM branch has been tagged (built)
-                            // need to explicitly provide name of correlated to DOCS-CSM branch
-                            sh "./verify-docs-csm-tag.sh ${env.DOCS_CSM_BRANCH}"
-                        }
-                    }
-                }
-                stage('Validate csm-vshasta-deploy branch') {
-                    steps {
-                        script {
-                            if ( ! jenkinsUtils.findJob("Cray-HPE", "csm-vshasta-deploy", env.CSM_VSHASTA_DEPLOY_BRANCH) ) {
-                                error("Branch ${env.CSM_VSHASTA_DEPLOY_BRANCH} in csm-vshasta-deploy repo does not exist")
-                            }
-                        }
+                    if ( ! jenkinsUtils.findJob("Cray-HPE", "csm-vshasta-deploy", env.CSM_VSHASTA_DEPLOY_BRANCH) ) {
```

Comment:
@phalseth-hpe This check is moved into 'Setup', because it's a very simple setup time check. The other check for docs-csm tag was replaced with `API Documentaion` stage, so I've eliminated entire `Pre-Release Validate` stage.